app/routes/jobs._index.tsx

```diff
@@ -242,6 +264,17 @@ export default function JobsPage() {
 
   return (
     <main>
+      <div className="flex items-center pb-4">
+        <Input
+          placeholder="Filter jobs by name..."
```

Comment:
Would be nice to have a clear search button, which can be done with

```suggestion
          placeholder="Filter jobs by name..."
          type="search"
          style={{ colorScheme: useTheme()[0] === "dark" ? "dark" : "light" }}
```

Suggestion also needs import of useTheme.